docs/Release_notes.md

```diff
@@ -1,6 +1,33 @@
 JPO-ODE Release Notes
 ----------------------------
 
+Version 2.1.0, released June 2024
+----------------------------------------
+### **Summary**
+The updates for the jpo-ode 2.1.0 release include several key improvements and fixes. These updates address issues with PSM and log offloading and enhance the continuous integration processes. The Kafka version has been upgraded and a bug related to log processing has been resolved. Nanoseconds are now trimmed from timestamps and 1609.2 headers are now stripped from unsigned messages. A submodule compatibility guide has been added, along with making the default SNMP protocol configurable. Configurable signing is now possible independently for Road Side Units (RSUs) and the SDX. The Dockerhub documentation now includes a link to the submodule compatibility guide. Maven plugin versions have been updated and the Kafka topic creation process has been improved. A timestamp precision bug has been fixed and the documentation has been revised for accuracy. Additionally, the NTCIP1218 msgRepeatOptions value is now set based on context and SnmpSession now reports failures to retrieve authoritative engine IDs only if a response is received. Finally, the TimDeleteController has been updated to log message deletion failures as errors.
```

Comment:
Technically if you are going to spell out RSU, it would be "Roadside Unit". You could make it "Road-Side Unit" if you want to make it similar to OBU with "On-Board Unit"

Reply:
Changed to `Road-Side Unit`